blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Upgrades to the REPL in v0.6"
+author: <a href="https://kristofferc.github.io/">Kristoffer Carlsson</a>
+---
+
+Since v0.3, Julia comes with a REPL completely written in Julia ([PR #6270](https://github.com/JuliaLang/julia/pull/6270)).
+It contains, among other features, code completion, history with reverse search, a rich set of keybindings which in addition are customizable (see the [Interacting with Julia](http://docs.julialang.org/en/stable/manual/interacting-with-julia/) part of the manual for REPL documentation).
+Even so, v0.6 of Julia comes with quite a few new visual tweaks and feature upgrades over v0.5.
+In this blog post, we look into some of these changes and customizations to the Julia REPL available in v0.6.
+
+## Prompt pasting
+
+It is quite common (as in the julia manual and in subsequent code blocks of this blog post) to write snippets of julia code like they were copied straight from a REPL session, for example:
+
+```jl
```

Comment:
Are you sure `jl` works? I thought it referred to something else. I could be wrong on that. Might be a good idea to use `julia` just in case though.